### PR TITLE
 added vite config and plugin for pwa

### PR DESCRIPTION
**Pull request by 0Clark30**

### CODE REVIEW

**Review by harleysherman — APPROVED** [review]

**Review by harleysherman — APPROVED** [review]

**Review by harleysherman — APPROVED** [review]

File: .eslintrc.cjs
Comment (0Clark30) [inline .eslintrc.cjs:1]: good

File: .gitignore
Comment (0Clark30) [inline .gitignore:1]: good

File: LICENSE
Comment (0Clark30) [inline LICENSE:1]: good

File: README.md
Comment (0Clark30) [inline README.md:1]: good

File: index.html
Comment (0Clark30) [inline index.html:1]: good

File: package-lock.json
Comment (0Clark30) [inline package-lock.json:1]: good

File: src/App.css
Comment (0Clark30) [inline src/App.css:1]: good

File: package.json
Comment (0Clark30) [inline package.json:1]: good

File: public/vite.svg
Comment (0Clark30) [inline public/vite.svg:1]: g

File: src/App.jsx
Comment (0Clark30) [inline src/App.jsx:1]: good

File: src/assets/react.svg
Comment (0Clark30) [inline src/assets/react.svg:1]: good

File: src/index.css
Comment (0Clark30) [inline src/index.css:1]: good

File: src/main.jsx
Comment (0Clark30) [inline src/main.jsx:1]: good

File: vite.config.js
Comment (0Clark30) [inline vite.config.js:1]: good

**Pull request closed without merging**